robotiq_s_model_articulated_gazebo/launch/robotiq_gripper_empty_world.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="limited" default="false"/>
+
+  <!-- startup simulated world --> 
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="world_name" default="worlds/empty.world"/>
+    <arg name="paused" value="false"/>
+  </include>
+
+  <!-- send robot to urdf to param serv -->
+  <include file="$(find robotiq_s_model_visualization)/launch/robotiq_gripper_upload.launch">
+  </include>
+
+  <!-- Start the publisher for the left hand (the only hand) -->
+  <node pkg="robot_state_publisher" type="state_publisher" name="robotiq_hands_l_hand_robot_state_publisher">
+    <param name="publish_frequency" type="double" value="50.0" />
+    <remap from="joint_states" to="/robotiq_hands/left_hand/joint_states" />
```

Comment:
We can add this pull request, and as an issue/feature upgrade make it allowable to work with any prefix.